apps/server/controlplane/src/server/world.py

```
@@ -27,6 +27,27 @@ def __init__(self, bedrock: ServerConsole, broker: EventPublisher) -> None:
 
+    def query_world_state(self) -> dict[str, str]:
+        """Query current time and weather from the Bedrock console and persist them."""
+        result: dict[str, str] = {}
+        for query in ("daytime", "day"):
+            try:
+                output = self.bedrock.send_and_read(["time", "query", query])
+                numbers = re.findall(r"-?\d+", output)
+                if numbers:
+                    result[query] = numbers[-1]
+            except Exception:
+                pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)